packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -52,6 +52,16 @@ const propTypes = {
    * Level of nesting for this item.
    */
   level: PropTypes.number,
+  /**
+   * @private
+   * The position of the item among its sibling items in the same group (subfolder).
+   */
+  posInSet: PropTypes.number,
```

Comment:
does this mean `position in set`? Can we rename this to `positionInSet` or `itemPosition` for clarity?

Reply:
I was torn between naming it `posInSet`, `ariaPosInSet`, or `positionInSet` as you suggested since this is a direct passthrough to `aria-posinset`. I'm not sure which is most appropriate - maybe `ariaPosInSet` since it matches the aria attribute?

Reply:
Oh if it's a direct passthrough, then yeah, `ariaPosInSet` sounds good!

Reply:
Updated both prop names: [eccff30](https://github.com/cerner/terra-framework/pull/2106/commits/eccff30d49e9d7cc6eae52c235f11dfb90dc086a)